docs/user-guide/src/version_support.md

```diff
@@ -57,6 +57,21 @@ Following table summarizes BMO release/test process:
 | v0.2          | EOL       |
 | v0.1          | EOL       |
 
+## Ironic-image
+
+Since `v23.1.0`, Ironic follows the semantic versioning scheme for its own
+release cycle, the same way as CAPM3 and IPAM. Currently, we have release-24.1, release-24.0
+and release-23.1 release branches for `v24.1.x` `v24.0.x` and `v23.1.x` release cycle
```

Comment:
Do you want to mention 25.0 and its release cycle as well?

Reply:
I put it in the table, but forgot the text part, as 25.0 came out just after initial version of the PR. Good catch, and fixed.
The text is now saying `two or three`  as we need to maintain any number needed for CAPM3 releases on Metal3, even though Ironic bugfix branch itself would not be maintained.